### PR TITLE
Adds sample configs

### LINKED ISSUE #22: Example stage/prod configs.
I'll need example stage and prod configs. If they are close to the same, one will suffice.

### PR DESCRIPTION
**Pull request by kitcambridge**
Closes #22.

I also added separate `router_ssl_{key, cert}` options, since I think we'll want to expose a TLS listener for WebSocket connections, but just a plaintext TCP listener for internal routing requests. Access to the routing port can be restricted via security groups.

@oremj r?

### CODE REVIEW

File: shared_settings.ini [hunk 1]
```diff
@@ -0,0 +1,48 @@
+; Settings shared by the connection and endpoint nodes. Overrides
+; should be specified in `autopush_settings.ini` and
+; `autoendpoint_settings.ini`.
+
+; Enable verbose logging for WebSocket connections and HTTP requests.
+; This should be disabled in production.
+debug = false
+
+; The public hostname. Defaults to the system hostname if unspecified.
+hostname =
```
Comment (oremj) [inline shared_settings.ini]: Will this pull the ec2 hostname, like simplepush?

File: autopush_settings.ini [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+; Settings for the internal routing listener, used to deliver routed
+; notifications to clients connected to this node.
+router_hostname =
+router_port = 8081
+router_ssl_key =
+router_ssl_cert =
+
+; The endpoint hostname and port, used to construct the push endpoint
+; URL for each registered channel.
+endpoint_hostname =
```
Comment (oremj) [inline autopush_settings.ini]: Can I get an example hostname.

File: autopush_settings.ini [hunk 3]
```diff
@@ -0,0 +1,17 @@
+; Connection node settings.
+
+[connection]
+; The port on which to listen for WebSocket connections.
+port = 8080
+
+; Settings for the internal routing listener, used to deliver routed
+; notifications to clients connected to this node.
+router_hostname =
```
Comment (oremj) [inline autopush_settings.ini]: Example hostname here too.

File: shared_settings.ini (hunk 1 above)
Comment (kitcambridge) [inline shared_settings.ini]: This just uses `socket.gethostname()`...so I think that'll return the internal hostname? I can clarify.

**Comment by oremj** [comment]
I just need clarification about what the different host names are referring to. Otherwise, lgtm.

File: shared_settings.ini [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+; exposed by the connection node uses separate keys, specified in the
+; `[connection]` section. If omitted, TLS will not be used.
+ssl_key =
+ssl_cert =
+
+; The push endpoint token encryption key. This should be identical for
+; connection and endpoint nodes in the same cluster; otherwise,
+; endpoints won't be able to decrypt tokens generated by the
+; connection node.
+crypto_key = i_CYcNKa2YXrF_7V1Y-2MFfoEl7b6KX55y_9uvOKfJQ=
```
Comment (oremj) [inline shared_settings.ini]: Need docs on how to generate a key.

File: configs/autopush_shared.ini [hunk 5]
```diff
@@ -0,0 +1,43 @@
+; Settings shared by the connection and endpoint nodes.
+
+; Enable verbose logging for WebSocket connections and HTTP requests.
+; This should be disabled in production.
+debug = false
```
Comment (oremj) [inline configs/autopush_shared.ini]: autopush: error: debug is a flag but is being set to 'false'

Getting that with `debug = false`.

**Pull request merged**